packages/app/obojobo-express/server/viewer/viewer_notification_state.js

```diff
@@ -0,0 +1,57 @@
+const db = oboRequire('server/db')
+
+function getStatus(id) {
+	return db.oneOrNone(
+		`
+		SELECT status FROM notification_status
+		WHERE
+			id = $[id]
+	`,
+		{
+			id
+		}
+	)
+}
+function getTitle(id) {
+	return db.oneOrNone(
+		`
+		SELECT title FROM notification_status
+		WHERE
+			id = $[id]
+	`,
+		{
+			id
+		}
+	)
+}
+function getText(id) {
+	return db.oneOrNone(
+		`
+		SELECT text FROM notification_status
+		WHERE
+			id = $[id]
+	`,
+		{
+			id
+		}
+	)
+}
```

Comment:
Don't create multiple functions for each individual field, just have a single `getNotifications(ids)` function to grab multiple notifications - pass it an array (could even be an array containing a single number) and handle results accordingly.